main.go

```diff
@@ -39,6 +29,17 @@ func main() {
 }
 
 func realMain(logger hclog.Logger) error {
+	var (
+		endpoint     = flag.String("endpoint", "/tmp/vault.sock", "path to socket on which to listen for driver gRPC calls")
+		debug        = flag.Bool("debug", false, "sets log to debug level")
+		selfVersion  = flag.Bool("version", false, "prints the version information")
+		vaultAddr    = flag.String("vault-addr", "https://127.0.0.1:8200", "default address for connecting to Vault")
+		vaultMount   = flag.String("vault-mount", "kubernetes", "default Vault mount path for Kubernetes authentication")
+		writeSecrets = flag.Bool("write-secrets", true, "write secrets directly to filesystem (true), or send secrets to CSI driver in gRPC response (false)")
+		healthAddr   = new(string)
+	)
+	flag.StringVar(healthAddr, "health_addr", "", "deprecated, please use --health-addr")
```

Comment:
Should this still have a default of `:8080`?

Reply:
I tested this and `flag.StringVar` immediately sets the contents of the pointer you pass it, so only the second call's default has an effect. Otherwise, the default just turns up in the usage help text and we only want usage to be the deprecated notice, so I've omitted it. This is 100% gross though and is why I've set a timeline for actually removing the flag value rather than leaving it around forever.